docs/integrations/amazon-aws/sqs.md

```diff
@@ -131,18 +135,13 @@ Namespace for **Amazon SQS** Service is **AWS/SQS**
 Login to Sumo Logic,  go to Manage Data > Logs > Fields. Search for the “queuename” field. If not present, create it. Learn how to create and manage fields [here](https://help.sumologic.com/Manage/Fields#manage-fields).
 
 ## Field Extraction Rule(s)
-
 Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
 
-**Rule Name**: AwsObservabilitySQSCloudTrailLogsFER
-
-**Applied at**: Ingest Time
-
-**Scope (Specific Data)**: account=* eventname eventsource "sqs.amazonaws.com"
-
-**Parse Expression**:
+* **Rule Name**: AwsObservabilitySQSCloudTrailLogsFER
+* **Applied at**: Ingest Time
+* **Scope (Specific Data)**: account=* eventname eventsource "sqs.amazonaws.com"
 
-```
+```sql title="Parse Expression"
```

Comment:
Parse expression should be outside code block. Just as it is on the main branch.

Reply:
Leave it as it was on the main branch.